toxcore/mono_time_test.cc

```diff
@@ -51,13 +51,16 @@ TEST(MonoTime, IsTimeoutReal)
     ASSERT_NE(mono_time, nullptr);
 
     uint64_t const start = mono_time_get(mono_time);
-    EXPECT_FALSE(mono_time_is_timeout(mono_time, start, 5));
+    EXPECT_FALSE(mono_time_is_timeout(mono_time, start, 5000));
```

Comment:
the timeout parameter is in seconds. is this intentional? (5000sec)

Reply:
Leftover from past test. Fixed.